src/token/src/canister/dip20_transactions.rs

```diff
@@ -686,4 +686,43 @@ mod tests {
         let canister = test_canister();
         canister.getTransaction(Nat::from(2));
     }
+
+    #[test]
+    fn get_user_transactions() {
+        let canister = test_canister();
+        const COUNT: usize = 5;
+        for _ in 1..COUNT {
```

Comment:
for _ in 1..5, can delete count as not used elsewhere

Reply:
true